OpenRA.Mods.Common/Widgets/Logic/Installation/DownloadPackageLogic.cs

```diff
@@ -259,10 +249,8 @@ void ShowDownloadDialog()
 						Log.Write("install", e.ToString());
 						onError("Online mirror is not available. Please install from an original disc.");
 					}
-				};
+				});
 
-				var updateMirrors = new Download(download.MirrorList, onDownloadProgress, onFetchMirrorsComplete);
-				cancelButton.OnClick = updateMirrors.CancelAsync;
 				retryButton.OnClick = ShowDownloadDialog;
```

Comment:
You shouldn't need to set this here again. Setting it inside the `downloadUrl` Action should be enough.